### DISCUSSION TITLE
How to get the old value in the `$watch` handler?

### DISCUSSION THREAD

**Discussion by northern-mindset** [alpinejs/alpine#1359] [1. Help]
Is it possible to get the old value in the `$watch` handler? And how?

**Comment by KevinBatdorf** [comment]
You'd have to store it outside and updated it within `$watch` 

Seems like a good PR though so you can pass it in:
```js
$watch('myVar', (value, oldValue) => {})
```
but currently that doesn't work. Here's the relevant code though: https://github.com/alpinejs/alpine/blob/master/src/component.js#L131
